guides/catalog/01 Searching and Rastering Catalog Products.ipynb

```diff
@@ -24,105 +21,19 @@
-    "\n",
-    "`Product.search()` is the entry point for searching products. It returns a query builder that you can use to refine your search and can iterate over to retrieve search results.\n",
-    "\n",
-    "We can count all products with some data before 2023 using `filter()`:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "13a9b0cb-9c12-4c36-b4dc-f82e9941cff8",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "search = Product.search().filter(p.start_datetime < \"2023-01-01\")\n",
-    "search.count()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "8da3bd7a-36de-4ac5-87c7-ce220964b18b",
-   "metadata": {},
-   "source": [
-    "You can apply multiple filters in succession to the same `Search` object. To restrict this search to products with data before 2023 and after 2000:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "15c80c17-46c9-4f83-a584-6ffc8c6a8e8d",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "search = search.filter(p.end_datetime > \"2000-01-01\")\n",
-    "search.count()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "b6055762-3d23-4c0f-a330-18292e4e8e5b",
-   "metadata": {},
-   "source": [
-    "All attributes are documented in the [Product API reference](https://docs.descarteslabs.com/descarteslabs/catalog/docs/product.html#descarteslabs.catalog.Product), which also spells out which ones can be used to filter or sort.\n",
-    "\n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "6161e85c-2ee2-4882-9bb1-ec9b644ab4a6",
-   "metadata": {
-    "tags": []
-   },
-   "source": [
-    "### Text search\n",
-    "Add text search to the mix using `find_text()`. This finds all products with “landsat” in the name or description:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "583669be-3f88-427d-910b-1c3e97ab9284",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "landsat_search = search.find_text(\"landsat\").limit(None)\n",
-    "for product in landsat_search:\n",
-    "    print(product)"
-   ]
-  },
   {
    "cell_type": "markdown",
    "id": "82aab414-37a6-4dbc-abb6-2bb70c636d1d",
    "metadata": {},
    "source": [
-    "### Lookup by id and object relationships\n",
-    "If you have a `Product ID` you can create that object directly with `Product.get()`:"
+    "## Catalog Products\n",
+    "The foundational class within `Descartes Labs Catalog` is a `Product`. Think of a `Product` as a collection of imagery from the same data source, such as [Sentinel-2](https://sentinel.esa.int/web/sentinel/missions/sentinel-2), but more specifically with the same formatting level, such as  [Sentinel-2 Level 2A](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-2-msi/product-types/level-2a).\n",
```

Comment:
Similar vein: The foundational class within the `descarteslabs.catalog` module is `Product`. ...